.goreleaser.yaml

```diff
@@ -67,6 +67,10 @@ nfpms:
       - apk
       - deb
       - rpm
+    overrides:
+      rpm:
+        dependencies:
+          - /bin/sh
```

Comment:
Does this override also replace any existing dependencies not explicitly listed here, or is the default empty?

Reply:
The default is empty but it would override it. But since this dependency is only necessary for RPM and not for deb, that was the way to go to specifically not add it to the deb package, because it breaks the installation there